### PR TITLE
Updating docs.

### PR DESCRIPTION
**Pull request by vishh**
cc @mvdan

### CODE REVIEW

File: docs/standalone.md [hunk 1]
```diff
@@ -104,17 +104,16 @@ I0911 13:42:03.657918   21041 heapster.go:58] Heapster version 0.17.0
...
-###Test it out
+
+### Test it out
 If you got debug information as before, means that the heapster is working normally.You can test it by calling its [RESTful API](https://github.com/kubernetes/heapster/blob/master/docs/model.md).  
 You can use some restful tools to test, or just run a `curl` command (_ heapster implements gzip encoding, so we need add gunzip in command "curl"_).  
 
-```shell
-curl http://0.0.0.0:8082/api/v1/model/stats/ | gunzip
-```
+	curl http://0.0.0.0:8082/api/v1/model/stats/ | gunzip
```
Comment (jimmidyson) [inline docs/standalone.md]: So does heapster compress every response, even if the client hasn't requested compression?

Comment (vishh) [inline docs/standalone.md]: Yes. That is the current state.

Comment (jimmidyson) [inline docs/standalone.md]: Is that just due to the potential response size? We could just let go-restful handle the content encoding negotiation & let the client choose?

Comment (vishh) [inline docs/standalone.md]: Yeah. The response size was the concern. It makes sense to let clients
handshake though.

Comment (jimmidyson) [inline docs/standalone.md]: OK let me put in a PR for that.

**Comment by vishh** [comment]
cc @mvdan

File: docs/standalone.md (hunk 1 above)
Comment (jimmidyson) [inline docs/standalone.md]: PR merged - can get rid of the `| gunzip` or add `-H "Accept-Encoding: gzip"` to the curl command.

**Comment by mvdan** [comment]
After my docs fixes PR, this PR needs rebasing.

File: docs/standalone.md (hunk 1 above)
Comment (vishh) [inline docs/standalone.md]: Thanks @jimmidyson :100: 
Updated docs.

**Comment by mvdan** [comment]
LGTM. Flaky test unrelated to this PR - investigating atm.

**Pull request merged**